src/commonMain/kotlin/fr/acinq/lightning/channel/InteractiveTx.kt

```diff
@@ -1178,10 +1181,10 @@ sealed class SpliceStatus {
         val localPushAmount: MilliSatoshi,
         val remotePushAmount: MilliSatoshi,
         val liquidityLease: LiquidityAds.Lease?,
-        val origins: List<Origin.PayToOpenOrigin>
+        val origins: List<Origin>
     ) : QuiescentSpliceStatus()
     /** The splice transaction has been negotiated, we're exchanging signatures. */
-    data class WaitingForSigs(val session: InteractiveTxSigningSession, val origins: List<Origin.PayToOpenOrigin>) : QuiescentSpliceStatus()
+    data class WaitingForSigs(val session: InteractiveTxSigningSession, val liquidityLease: LiquidityAds.Lease?, val origins: List<Origin>) : QuiescentSpliceStatus()
```

Comment:
Note to self: see if we can get rid of the liquidity lease here and in `InProgress` to simplify serialization.

Reply:
We can't easily get rid of this, otherwise we won't be able to store the lease in the DB, which is useful, even though it means we'll need to handle backwards-compatibility if the format changes...